README.md

```diff
@@ -490,8 +489,15 @@ Resolves to:
 
   - If `expressSession` is set to `true`, it will use the configuration below:
 
+    - Roosevelt uses [memorystore](https://github.com/roccomuso/memorystore) as the default `express-session` store.
+
+    - To use the default store provided by `express-session`, use `"store": "memorystore"`
```

Comment:
change this from `"store": "memorystore"` to `"store": "express-session-default"`